### DISCUSSION TITLE
Syncing issue with server/desktop version conflict

### DISCUSSION THREAD

**Discussion by root-hal9000** [TriliumNext/Trilium#4581] [Q&A] [answered]
So I just learned about Trilium and started by running it in a docker container (on my Synology NAS) with the latest tag, which gave me version  0.63.1-beta. I played around with it, started organizing some of my scattered notes from .txt files etc, and I love it.
BUT then I went to install the desktop app (running Linux) and sync, and I am not able to, because of the sync version:
Server is 0.63.1-beta with sync version 32
Desktop  is 0.62.5 with sync version 31
So it throws an error when I try to start the desktop app selecting sync from server. Is there any way I can avoid starting from scratch? I got carried away and already added a decent amount of notes on the web version before this came up. What are my options?

1.  wait for desktop version to catch up? I am running kubuntu and it shows available as Flatpak (which I installed) and Snap, both as version 0.62.5 
2. somehow downgrade the version I am running on docker? Is there a way to do this without throwing everything away?
3. start over? can i export stuff out in a way that will be compatible if I start the server from scratch with a similar sync version?
4. perhaps most important: how can i prevent this issue going forward, since I assume I will be updating as new releases come out?

I appreciate any tips for how to handle this... Thanks to the creators, I am loving it so far, it's just what I needed, and I can already see how it will save me so much time and sanity from dealing with my scared notes and bookmarks :)

**Comment by hulmgulm** [comment, marked as answer]
You can download the 0.63.1-beta app for desktop here: https://github.com/zadam/trilium/releases/tag/v0.63.1-beta

**Reply by root-hal9000** [reply]
thank you! Got it synced. Not sure why i didn't just do that in the first place
